src/ln_wallet_endpoint.py

```diff
@@ -199,6 +199,30 @@ def channel_opener():
             flash(Markup(e.args[0]), 'danger')
     return render_template('lightning/channel_opener.html', funds_dict=LnRpc().list_funds())
 
+@ln_wallet.route('/peer_management', methods=['GET', 'POST'])
+@roles_accepted(Role.ROLE_ADMIN)
+def peer_management():
+    rpc = LnRpc()
+    if request.method == 'POST':
+        if request.form['form-name'] == 'close_peer_form':
+            peer_id = request.form["peerId"]
```

Comment:
```suggestion
            peer_id = request.form['peerId']
```
keep quotes consistent

Reply:
changed per above.